### DISCUSSION TITLE
Best practices for storing and processing data

### DISCUSSION THREAD

**Discussion by samevision** [xyflow/xyflow#2543] [Q&A]
Hello,

are there examples of scalable best practices for storing and processing data in DAG using react flow?

Best

**Comment by mdugue** [comment]
hey @samevision  , I just stumbled upon your question as I am currently also looking for using react-flow for some "no code DAG" app. Did you find any helpful resources therefore?

(This is a copy of the message I also just sent to you on discord. Not sure what the best platform is to discuss this topic)
